src/FirstAssignment/CarbonFootprint.java

```diff
@@ -0,0 +1,45 @@
+package FirstAssignment;
+
+import java.util.Scanner;
+
+import javax.mail.Folder;
+import javax.mail.MessagingException;
+import javax.mail.Store;
+
+public class CarbonFootprint {
+	public static final double CARBON_EMISSION_OF_INBOX_MAIL = 0.3;
+	public static final double CARBON_EMISSION_OF_SPAM_MAIL = 0.7;
+	public static final double CARBON_EMISSION_OF_SENT_MAIL = 0.4;
+
+	public static int inboxMails;
+	public static int spamMails;
+	public static int sentMails;
+
+	public static void main(String[] args) throws MessagingException {
+		Scanner scan = new Scanner(System.in);
+		System.out.println("Enter email acccount details");
+		System.out.println("Enter email address");
+		String emailAddress = scan.next();
+		System.out.println("Enter 12 letter app key without any space");
+		String appKey = scan.next();
+
+		Store storeInstance = ConnectionMailServer.getConnectionStoreInstance(emailAddress, appKey);
+
+		inboxMails = storeInstance.getFolder("Inbox").getMessageCount();
+		spamMails = storeInstance.getFolder("Inbox").getMessageCount();

```

Comment:
Why are we obtaining the message count of the Inbox for spam emails, and likewise for sent mails?